cdhweb/pages/snippets.py

```diff
@@ -233,7 +233,7 @@ class Meta:
 
     panels = [
         InlinePanel("contact_links", label="Contact Links", min_num=1),
-        InlinePanel("social_media_links", label="Social Media Links", max_num=4),
+        InlinePanel("social_media_links", label="Social Media Links"),
```

Comment:
@sarahframe I found I couldn't add the new link, and this was why. I figure it might be best to just remove the limit to stop this accidentally happening again...? Thoughts?

Reply:
we could change it to 6

Reply:
ok, if that's your preference. done :)